src/App.js

```diff
@@ -1,37 +1,69 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import Cards from './pages/Cards'
 
+const recipesData = [
+  {
+    id: 1,
+    name: 'Roasted vegetables',
+    products: ['potatoes', 'mushrooms', 'zucchini', 'paprika', 'spices'],
+    image: 'https://i.picsum.photos/id/1037/200/200.jpg',
+    instruction:
+      'Cut vegetables in slices. Add some spices and put them in the oven at 180° for about 45 minutes.',
+    isBookmarked: false,
+  },
+  {
+    id: 2,
+    name: 'Roasted potatoes and mushrooms  ',
+    products: ['potatoes', 'mushrooms', 'spices'],
+    image: 'https://i.picsum.photos/id/1037/200/200.jpg',
+    instruction:
+      'Cut vegetables in slices. Add some spices and put them in the oven at 180° for about 45 minutes.',
+    isBookmarked: false,
+  },
+  {
+    id: 3,
+    name: 'Roasted potatoes ',
+    products: ['potatoes'],
+    image: 'https://i.picsum.photos/id/1037/200/200.jpg',
+    instruction:
+      'Cut potatoes in slices. Add some spices and put them in the oven at 180° for about 45 minutes.',
+    isBookmarked: false,
+  },
+]
+
 export default function App() {
-  const recipes = [
-    {
-      id: 1,
-      name: 'Roasted vegetables',
-      products: ['potatoes', 'mushrooms', 'zucchini', 'paprika', 'spices'],
-      image: 'https://i.picsum.photos/id/1037/200/200.jpg',
-      instruction:
-        'Cut vegetables in slices. Add some spices and put them in the oven at 180° for about 45 minutes.',
-    },
-    {
-      id: 2,
-      name: 'Roasted potatoes and mushrooms  ',
-      products: ['potatoes', 'mushrooms', 'spices'],
-      image: 'https://i.picsum.photos/id/1037/200/200.jpg',
-      instruction:
-        'Cut vegetables in slices. Add some spices and put them in the oven at 180° for about 45 minutes.',
-    },
-    {
-      id: 3,
-      name: 'Roasted potatoes ',
-      products: ['potatoes'],
-      image: 'https://i.picsum.photos/id/1037/200/200.jpg',
-      instruction:
-        'Cut potatoes in slices. Add some spices and put them in the oven at 180° for about 45 minutes.',
-    },
-  ]
+  const [recipes, setRecipes] = useState(getFromLocal('recipes') || recipesData)
+
```

Comment:
There is an opportunity to refacter `getFromLocal` and `setToLocal` into their own file, maybe `services.js`